motion/motionprocessor.go

```diff
@@ -159,6 +187,41 @@ func (mp *MotionProcessor) Process(rawFrame []byte) error {
 	return nil
 }
 
+func (mp *MotionProcessor) cameraTempControll(frame *cptvframe.Frame) {
+	if mp.finishedWarmup {
+		if frame.Status.TimeOn-frame.Status.LastFFCTime > 2*time.Minute &&
```

Comment:
might be nicer to put the if statement in a function e.g.
`func () tempRequiresSaving() bool`